src/main/common/filter.c

```diff
@@ -57,8 +57,7 @@ float pt1FilterGainFromDelay(float delay, float dT)
         return 1.0f; // gain = 1 means no filtering
     }
 
-    const float cutoffHz = 1.0f / (2.0f * M_PIf * delay);
```

Comment:
Is this really necessary? IMO generated code would be identical.
At least leave cutoff calculation as comment